.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+  "name": "tldr-pages",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {},
+    "ghcr.io/devcontainers/features/python:1": {}
+  },
+
+  "privileged": true,
+
+  "onCreateCommand": {
+    "update-apt-packages": "sudo apt update && sudo apt upgrade -y",
```

Comment:
This command updates apt-packages, this command takes some time, but with prebuilds you can save a few minutes (in prebuilds, all packages will be pre-installed)